src/Fixer/Comment/NoEmptyCommentFixer.php

```diff
@@ -149,7 +149,7 @@ private function getLineBreakCount(Tokens $tokens, int $whiteStart, int $whiteEn
 
     private function isEmptyComment(string $content): bool
     {
-        static $mapper = [
+        $mapper = [
```

Comment:
in previous file (src/Fixer/ClassNotation/OrderedTypesFixer.php), you inlined `$glue` with `[...definition][$index]`.
why keeping the variable here and not inline it on line 160?

i wonder which way to go, but probably one is enough to follow (no need for 2 approaches)